dev/PushNotifications/externs.h

```diff
@@ -21,3 +21,26 @@ inline HRESULT GetCurrentProcessPath(wil::unique_cotaskmem_string& processName)
     return wil::GetModuleFileNameExW(GetCurrentProcess(), nullptr, processName);
 };
 
+inline HRESULT GetAppUserModelId(wil::unique_cotaskmem_string& appUserModelId)
+{
+    wchar_t appId[APPLICATION_USER_MODEL_ID_MAX_LENGTH] = {};
+    UINT32 appIdSize{ ARRAYSIZE(appId) };
+
+    GetCurrentApplicationUserModelId(&appIdSize, appId);
+
+    appUserModelId = wil::make_unique_string<wil::unique_cotaskmem_string>(appId);
+
+    return S_OK;
+}
+
+inline const std::string ConvertProcessNameToUtf8String(std::wstring processName)
```

Comment:
@danielayala94 Already introduced a function for this. We should not copy paste but introduce Utility Helpers with shared common code.

Reply:
Currently moved all this into externs.h. Will create a utility to be used between LRP and PushNotifications. bug: https://github.com/microsoft/WindowsAppSDK/issues/1481